postfix/datadog_checks/postfix/config_models/shared.py

```diff
@@ -0,0 +1,44 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, root_validator, validator
+
+from datadog_checks.base.utils.functions import identity
+from datadog_checks.base.utils.models import validation
+
+from . import defaults, validators
+
+
+class SharedConfig(BaseModel):
+    class Config:
+        allow_mutation = False
+
+    postfix_user: str
```

Comment:
Not required, default is `root` https://github.com/DataDog/integrations-core/blob/231435d118b1d4ed4df00137e93816894913bdc2/postfix/datadog_checks/postfix/postfix.py#L186